rfcs/py_3.md

```diff
@@ -0,0 +1,54 @@
+# RFC 69: Switch web-platform-tests to Python 3
+
+## Summary
+
+This RFC sets out the overall plan and timetable for the Python
+2->Python 3 transition. It is anticipated that further details on the
+transition will be set out in other RFCs as appropriate.
+
+## Details
+
+ 1. **November / December 2020** XXX add actual date - Python 3 becomes default on GitHub CI
```

Comment:
Should the 'XXX add actual date' be removed?

Reply:
At a guess, first attempt at this will likely happen the week of the 23rd. The handlers change RFC can be landed on the 17th, but that's the week of BlinkOn for us so I don't want to do the Python 3 switchover at the same time (and I'd like to give the handlers time to bake as well).

Reply:
I updated the document to list that date. I think if we miss the week of the 23rd for whatever reason we should consider pushing the other dates back too.